docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:
+    gateway:
+        image: gateway:latest
+        environment:
+                - PORT=8080
+        network_mode: "host"
```

Comment:
is there any requirement for running in "host" network_mode?

Reply:
It is designed this way to simplify mapping between host / container ports.

Reply:
But we would appreciate if you could contribute with us to create a ready-to-deploy version. :)

Reply:
of course I will :)